### ISSUE TITLE
Update working group plan to better reflect open source

### ISSUE THREAD

**Issue by awentzel** [Microsoft/fast-dna#31]
@awentzel commented on [Wed Jan 17 2018](https://github.com/Microsoft/fast/issues/14)

Though it might be better to rename or recreate as fast-wg-a11y@ and fast-wg-security@

- Evaluate the notes below to determine what changes should be made to better align OSS community around **security** and **accessibility**
- Update policy documentation
- Update email groups, descriptions, members, owners.

It is recommended that the Steering Committee establish a security working group of collaborators with recognized security experts tasked with reviewing security related pull requests and determining an appropriate review process.

### Long-term support working group roadmap

The LTS WG is expected to establish a regular and predictable cadence of LTS Releases. To this end, the LTS WG must maintain and regularly publish a clear roadmap that outlines the priorities and milestones for upcoming LTS releases. The goal of the roadmap is to help guide the project's evolution as opposed to constraining it.

### Working groups

Working groups are autonomous groups of collaborators chartered by the SC to oversee specific aspects of the project. Working groups can be formed at any time but require ratification by the TSC.

Every working group has a charter that details its area of responsibility. Once approved by the SC, the working group becomes solely responsible for items detailed in that charter.

When applicable, the charter should indicate the conditions under which the working group's activity can be considered to be complete, and the WG dissolved. A working group can, at any time, request that the SC dissolve the working group by opening an issue requesting dissolution.

The SC may revoke the working groups charter at any time following the decision-making process defined in the SC charter.

Proposals to create a new working group should be made either by opening an issue with the draft charter for the new working group, then putting that on the SC agenda.

**Comment by awentzel** [comment]
This is in future preparation. It's not needed now, but, when the codebase matures and security or accessibility related tasks become commonplace we should implement this feature.
